File: Web/RMA-Web-Shell/projects/shared-models-lib/src/lib/common/ruleItem.ts
export class RuleItem {
    id: number;
    createdBy: string;
    modifiedBy: string;
    createdDate: Date;
    modifiedDate: Date;
    isDeleted: boolean;

    name: string; // TODO not sure what this is used for
    itemId: number;
    ruleId: number;
    ruleConfiguration: string;
}

File: Common/RMA.Common/Entities/RuleItem.cs
using System;

namespace RMA.Common.Entities
{
    public class RuleItem
    {
        public int Id { get; set; }
        public bool IsDeleted { get; set; }
        public string CreatedBy { get; set; }
        public DateTime CreatedDate { get; set; }
        public string ModifiedBy { get; set; }
        public DateTime ModifiedDate { get; set; }

        public int ItemId { get; set; }
        public int RuleId { get; set; }
        public string RuleConfiguration { get; set; }
    }
}